Assets/Scripts/UI/PauseMenuUI.cs

```diff
@@ -50,7 +60,12 @@ public void Pause()
         pauseMenuUI.SetActive(true);
         Time.timeScale = 0f;
         gameIsPaused = true;
+        
+        playerInput.DisableInput();
         playerInput.DisableAbilityInput();
+
+        prevCursorVisibility = Cursor.visible;
+        prevCursorLockMode = Cursor.lockState;
```

Comment:
While testing this I found that the issue still exists so I swapped reading the previous cursor state and the assignments of the new cursor state.

Reply:
Ah sorry it did not work. I did not see any issues on my end. Weird fix though, I don't see any issues with the logic.